PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -441,6 +453,10 @@ public final class DefaultControlsViewController: ContentControlsViewController
         }
         compasDirectionView.transform = uiProps.compasDirectionViewTransform
         cameraPanGestureRecognizer.isEnabled = uiProps.cameraPanGestureIsEnabled
+        if #available(iOS 11.0, *) {} else {
```

Comment:
Bad practice - please rewrite it without  `{ }`

Reply:
@AndriiMoskvin Sure.